src/coreclr/src/inc/eventtracebase.h

```diff
@@ -65,6 +65,154 @@ enum EtwThreadFlags
     kEtwThreadFlagThreadPoolWorker =  0x00000004,
 };
 
+#if defined(FEATURE_PERFTRACING)
```

Comment:
Is there a reason this whole block of code needed to move? It makes it harder to see the changes when they are obscured in the movement of larger blocks.

Reply:
Sorry, this had to move :( It's because the class XplatEventLoggerConfiguration is not defined on Windows currently.

Reply:
What if you just make it...
 
````
250      #endif //defined(HOST_UNIX) && (defined(FEATURE_EVENT_TRACE) || defined(FEATURE_EVENTSOURCE_XPLAT))
251-371  existing XplatEventLoggerConfiguration
372      #if defined(HOST_UNIX) && (defined(FEATURE_EVENT_TRACE) || defined(FEATURE_EVENTSOURCE_XPLAT))

More broadly I'd usually favor leaving code elements where they are and adjusting defines (or other attributes) around them unless it is very important for understanding that all similar code elements be contiguous.